ajax/libs/caver-js/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "caver-js",
+  "title": "JavaScript API Library for Klaytn",
+  "description": "caver-js is a JavaScript API library that allows developers to interact with a Klaytn node",
+  "author": {
+    "name": "Klaytn Team",
+    "url": "https://www.klaytn.com"
+  },
+  "license": "LGPL",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/klaytn/caver-js.git"
+  },
+  "homepage": "https://github.com/klaytn/caver-js",
+  "npmName": "caver-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

```suggestion
        "*"
```

there are no subfolder in dist